composer.json

```diff
@@ -19,7 +19,8 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "ext-libxml": "*"
+        "ext-libxml": "*",
+         "psr/http-message": "~1.0"
```

Comment:
I guess the psr/http-message is only for upload for psr-7 and we could add that to require-dev / add in suggest.